frontend/sass/style.scss

```diff
@@ -0,0 +1,22 @@
+@use "sass:map";
+
+@use "./abstracts/" as abstracts;
+@use "./base/" as base;
+@use "./components/" as components;
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
```

Comment:
Katoitko saako näitä toimimaan tuolla nextin avulla? `import { Roboto } from 'next/font/google'`

Reply:
En hoksannut testata, määpä katon saanko sen tuolla miten toimimaan.

Reply:
Jos käyttäjä pistetään lataamaan google fontsista rikotaan myös gdprää https://www.theregister.com/2022/01/31/website_fine_google_fonts_gdpr/